static/js/publisher/release/reducers/options.js

```diff
@@ -0,0 +1,3 @@
+export default function(state = {}) {
```

Comment:
Would be good to name this function for consistency. And some comment would be helpful to mention that currently these are only set in initial state.